### PR TITLE
Feature/da 8 backend repository setup

### PR DESCRIPTION
**Pull request by ffarlenvargas**
Add backend repository using NestJS + Serverless Framework + PrismaJS + GraphQL

Add schema.prisma using Prisma introspect functionality to get the DB structure.

Add advertiser module as example

Add advertiser test as example

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+*.sublime-workspace
+
+# IDE - VSCode
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+
+# These are some examples of commonly ignored file patterns.
```
Comment (ff-marcela-conejo) [inline .gitignore]: Can we remove this comment?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      "^.+\\.(t|j)s$": "ts-jest"
+    },
+    "collectCoverageFrom": [
+      "**/*.(t|j)s"
+    ],
+    "coverageDirectory": "../coverage",
+    "testEnvironment": "node"
+  },
+  "engines": {
+    "node": ">=18.20.4"
```
Comment (ff-marcela-conejo) [inline package.json:88]: Can we go with the newest version? Just for future-proofing

**Review by ff-marcela-conejo — APPROVED** [review]
Just a couple of small comments :)

File: package.json (hunk 2 above)
Comment (ffarlenvargas) [reply package.json:88]: Yes, it is possible. I changed it to 20.18.0, which is the newest version of node js supported by AWS lambda.
I think that if we use a newer version, we can have compatibility issues.

Please correct me if I am wrong.

This is the documentation that I reviewed.
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

File: .gitignore (hunk 1 above)
Comment (ffarlenvargas) [reply .gitignore]: Done 🚀

File: package-lock.json
Comment (ff-marcela-conejo) [inline package-lock.json:1]: Are we using yarn here? Just wondering if we need this file

**Review by ff-marcela-conejo — APPROVED** [review]

File: package-lock.json
Comment (ffarlenvargas) [reply package-lock.json:1]: I forgot to use yarn lol

Comment (ffarlenvargas) [reply package-lock.json:1]: Done @ff-marcela-conejo I migrated the project from npm to yarn

**Review by ff-marcela-conejo — APPROVED** [review]

**Pull request merged**